### DISCUSSION TITLE
Header's ActionComponent events are not working

### DISCUSSION THREAD

**Discussion by maqeelqureshi** [infinitered/ignite#2322] [Q&A] [answered]
I am trying to add animated drawer button in header component using LeftActionComponent property. 

Button in header is rendering properly but it's onPress event is not working.  How will this work?

         import { Button, Header } from "../../components"
           <Header
                titleTx="screen1.title"
                LeftActionComponent={
                     <Button
                              text="drawer"
                              onPress={() => {
                                      console.log("drawer clicked")  // This never prints
                              }}
                     ></Button>

                // <DrawerIconButton
                //   onPress={() => {
                //     toggleDrawer()
                //   }}
                //   {...{ open, progress }}
                // />
              } 
          >
         </Header>

**Comment by ryskin** [comment]
Is the sample you commented is working?

**Reply by maqeelqureshi** [reply]
No, its not working.

**Reply by frankcalise** [reply]
Hey this is actually a bug within header, I'm going to reopen your issue and just describe it more to add details. Thanks for the report!

**Comment by maqeelqureshi** [comment, marked as answer]
frankcalise suggestion of setting `zIndex: 2 to the $container style` worked for me.

ref#https://github.com/infinitered/ignite/issues/2320
